static/src/stylesheets/layout/new-header/_top-bar-dropdown.scss

```diff
@@ -8,52 +8,64 @@
 .dropdown-menu {
     display: none;
     position: absolute;
-    top: 29px; //visually this sits right under the text
-    left: -($gs-gutter / 2);
-    width: $gs-column-width * 3;
+    top: $gs-baseline * 2 + $gs-baseline / 2;
+    right: -8px;
+    width: $gs-column-width * 3 + $gs-gutter;
     background-color: #ffffff;
-    border-radius: 3px;
+    border-radius: $gs-baseline / 4;
     padding: $gs-baseline / 2 0;
     margin: 0;
-    box-shadow: 0 0 5px 0 rgba(0, 0, 0, .25);
+    box-shadow: 0 0 0 1px rgba(0, 0, 0, .1);
+    z-index: $zindex-main-menu;
 
     // triangle
     &:before,
     &:after {
         content: '';
         position: absolute;
+        right: $gs-gutter;
+        transform: translateX(50%) translateY(-100%);
     }
 
     &:before {
-        top: -( $gs-baseline / 2 + 1);
-        left: $gs-gutter - 2;
-        border-bottom: $gs-baseline / 2 + 1 solid $neutral-5;
-        border-left: $gs-baseline / 2 + 1 solid transparent;
-        border-right: $gs-baseline / 2 + 1 solid transparent;
+        top: -1px;
+        border-bottom: $gs-baseline / 2 + $gs-baseline / 4 solid rgba(0, 0, 0, .1);
+        border-left: $gs-baseline / 2 + $gs-baseline / 4 solid transparent;
+        border-right: $gs-baseline / 2 + $gs-baseline / 4 solid transparent;
     }
 
     &:after {
-        top: -( $gs-baseline / 2);
-        left: $gs-gutter - 1;
-        border-bottom: $gs-baseline / 2 solid #ffffff;
-        border-left: $gs-baseline / 2 solid transparent;
-        border-right: $gs-baseline / 2 solid transparent;
+        top: 0;
+        border-bottom: $gs-baseline / 2 + $gs-baseline / 4 - 1 solid #ffffff;
+        border-left: $gs-baseline / 2 + $gs-baseline / 4 - 1 solid transparent;
+        border-right: $gs-baseline / 2 + $gs-baseline / 4 - 1 solid transparent;
     }
 }
 
 .dropdown-menu--open {
     display: block;
+
+    & ~ .top-bar__item--dropdown {
```

Comment:
Should this happen for the fallback as well?

Reply:
yes! Good spot. Checking it non-JS helped me spot a font weight issue too.